src/services/tracing.ts

```diff
@@ -194,18 +196,31 @@ export const addEventToCurrentSpan = (
   }
 }
 
+export const recordException = (span: Span, exception: Exception, level?: ErrorLevel) => {
+  const errorLevel = level || exception["level"] || "warn"
+  span.setAttribute("error.level", errorLevel)
```

Comment:
We should set `error.level` on all spans if it exists (not just critical). That would be a great way to search on honeycomb.

Reply:
do you mean setStatus to error? because `const errorLevel = level || exception["level"] || "warn"` assign `warns` if exception does not have a level

Reply:
I did the update and also changed the default to `info` https://github.com/GaloyMoney/galoy/pull/1099/commits/807ca0979bba7e46574b50e350a5ca226d5877d1